cmd/xl-v1-object.go

```diff
@@ -295,7 +295,13 @@ func (xl xlObjects) getObject(ctx context.Context, bucket, object string, startO
 		// we return from this function.
 		closeBitrotReaders(readers)
 		if err != nil {
-			return toObjectErr(err, bucket, object)
+			if decodeHealErr, ok := err.(*errDecodeHealRequired); ok {
+				xl.addPartialUpload(bucket, object)
```

Comment:
Could you rename addPartialUpload() to something more generic ? partialUpload was meant for objects that was successfully upload (write quorum) but still not present in all disks

Reply:
How about renaming to `healUpload()`? Currently `xl.mrfUploadCh` accepts type `partialUpload`. Do I need to change names everywhere or just to this method?